frontend/src/components/housing/submission/SubmissionForm.vue

```diff
@@ -218,12 +218,8 @@ const onSubmit = async (values: any) => {
 };
 
 function updateLocationAddress(values: any, setFieldValue?: Function) {
-  const locationAddress = [];
-  if (values.streetAddress?.length) locationAddress.push(values.streetAddress);
-  if (values.locality?.length) locationAddress.push(values.locality);
-  if (values.province?.length) locationAddress.push(values.province);
+  const locationAddressStr = concatenateAddress(values);
```

Comment:
See comment in the utils function about `concatenateAddress` about passing three arguments instead of a single data object.